app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,21 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
```

Comment:
`But to have better code design, you should implement the abstract classes Elf, Dwarf and Player, where you can put the common logic.` - from task description
You should also inherit from ABS here